prover/lang/kore-lang.md

```diff
@@ -135,11 +137,13 @@ is to be used for generating fresh variables. *The second variety must be used
 only in this scenario*.
 
 ```k
-  syntax Variable ::= VariableName "{" Sort "}" [klabel(sortedVariable)]
+  syntax Variable    ::= VariableName  "{" Sort "}"  [klabel(sortedVariable)]
+  syntax SetVariable ::= VariableName "{{" Sort "}}" [klabel(sortedSetVariable)]
```

Comment:
Should we be moving away from having the sort attached to the variable if we are moving towards AML?
Maybe we want: `syntax SetVariable ::= "#" VariableName`, and `axiom \implies(#X, Int)` as a local declaration (or part of the claim).

Reply:
Will this break any existing tests? It's definitely a change that won't affect anything for the Coq translation though

Reply:
Since we don't use set variables yet, I don't think anything would break? It should be `\typeof(#X, Int)` not implies.

Reply:
Whoops sorry, I thought you were talking about all variables. I'll fix this for set variables.